### PR TITLE
Lesson_03: Add project for making basic soup

### PR DESCRIPTION
**Pull request by Xanchan**
Project with basic soup maker added for review
Signed-off-by: Oleksandr Siruk <alexsirook@gmail.com>

### CODE REVIEW

File: 03-make/make_soup/makefile [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/make
```
Comment (yekovalyov) [inline 03-make/make_soup/makefile]: Is it work without -f option?

File: 03-make/make_soup/makefile [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	echo "water boiled"
+
+ingridients.o:  get_ingridients.o prepare_ingridients.o
+	echo
+	echo "ingridients for soup are ready"
+
+get_ingridients.o:
+	echo
+	echo "get ingridients"
+	cp -a ${food-market}/. ${table}/
```
Comment (yekovalyov) [inline 03-make/make_soup/makefile]: Are /. and trailing / required?

File: 03-make/make_soup/makefile (hunk 1 above)
Comment (Xanchan) [reply 03-make/make_soup/makefile]: yup. Tested it and everything was fine

File: 03-make/make_soup/makefile (hunk 2 above)
Comment (Xanchan) [reply 03-make/make_soup/makefile]: as i read it's a special cp requirement for this case and it was failing without "/."

**Pull request merged**